shotover-proxy/src/frame/cassandra.rs

```diff
@@ -286,6 +286,18 @@ pub enum CassandraOperation {
 }
 
 impl CassandraOperation {
+    /// Return all queries contained within CassandaOperation::Query and CassandraOperation::Batch
+    pub fn queries(&mut self) -> std::slice::IterMut<Statement> {
+        match self {
+            CassandraOperation::Query {
+                query: CQL::Parsed(query),
+                ..
+            } => query.iter_mut(),
+            // TODO: Return CassandraOperation::Batch queries once we add BATCH parsing to cassandra-protocol
```

Comment:
Are all the TODOs intended to be done in the follow up PRs you mention?

Reply:
This is the only new TODO and yes I intend to do it in a follow up PR

Reply:
Oh true, the other one is just whitespace.